backend/app/assets/javascripts/spree/backend/taxon_autocomplete.js

```diff
@@ -1,30 +1,20 @@
-'use strict';
+$.fn.taxonAutocomplete = function () {
+  'use strict';
 
-var set_taxon_select = function(){
-  function formatTaxon(taxon) {
-    return Select2.util.escapeMarkup(taxon.pretty_name);
-  }
-
-  if ($('#product_taxon_ids').length > 0) {
-    $('#product_taxon_ids').select2({
+  this.select2({
       placeholder: Spree.translations.taxon_placeholder,
       multiple: true,
       initSelection: function (element, callback) {
-        Spree.ajax({
-          type: "GET",
-          url: Spree.routes.taxons_search,
-          data: {
-            ids: element.val(),
-            without_children: true
-          },
-          success: function (data) {
-            callback(data['taxons']);
-          }
+        var url = Spree.url(Spree.routes.taxons_search, {
```

Comment:
Actually just noticed this. [Spree.url is deprecated](https://github.com/solidusio/solidus/blob/master/core/app/assets/javascripts/spree.js.coffee.erb#L18). Is there a reason this was changed?